src/routes/__root.tsx

```diff
@@ -0,0 +1,26 @@
+import { createRootRoute, Outlet } from '@tanstack/react-router';
+import React, { Suspense } from 'react';
+import { SideNavigation } from '../components/SideNavigation';
+
+const TanStackRouterDevtools =
+  process.env.NODE_ENV === 'production'
+    ? () => null
+    : React.lazy(() =>
+        import('@tanstack/router-devtools').then(res => ({
+          default: res.TanStackRouterDevtools,
+        }))
+      );
+
+export const Route = createRootRoute({
+  component: () => (
+    <>
+      <SideNavigation />
+      <div className="bg-gray-50">
+        <Outlet />
+        <Suspense>
```

Comment:
doesn't it need a `fallback` ? @aromko

Reply:
Hm. I just copied it. According to the docs the default is fallback={null} so I think it's ok. It's just for the dev tools.

Reply:
ah okay understand